src/arch/bsdsocketinterf.cpp

```diff
@@ -137,6 +137,34 @@ int CBSDSocketInterface::sendDataOnTCP(TSocketDescriptor paSockD, const char* pa
   return nRetVal;
 }
 
+int CBSDSocketInterface::handleError(int nRetVal, int err, const char* msg) {
```

Comment:
As this is a private helper function I find it typically nicer to have them on the bottom of the file.